app/assets/stylesheets/ama_layout/layout_components/accordions.scss

```diff
@@ -76,6 +76,105 @@
   }
 }
 
+.accordion-multi-state{
+  @extend .accordion;
+  margin-bottom: $base-margin;
+  background: transparent;
+
+  .accordion-item{
+    margin-bottom: $base-margin;
+  }
+
+  &__header{
+    @extend .accordion__header;
+    background: transparent;
+    position: relative;
+  }
+
+  &__heading-bg{
+    @extend .accordion__heading;
+    padding: 0.63rem 1rem !important;
+    width: calc(100% - 20px);
+
+    &--active{
+      @extend .accordion-multi-state__heading-bg;
+      background: $brand-blue-light;
+    }
+
+    &--inactive{
+      @extend .accordion-multi-state__heading-bg;
+      background: $stone;
+
+      .accordion-multi-state__heading{
+        color: $slate;
+      }
+    }
+
+    &--complete{
+      @extend .accordion-multi-state__heading-bg;
+      background: $brand-green;
+    }
+
+    &--error{
+      @extend .accordion-multi-state__heading-bg;
+      background: $brand-red;
+    }
+  }
+
+  &__heading{
+    @extend .accordion__heading;
+    margin-left: 10px;
+    font-size: 1rem;
```

Comment:
Yes, we need it there so that it stays that font size no matter what screen size. Otherwise it gets too big on mobile.